src/System.Management.Automation/engine/Modules/GetModuleCommand.cs

```diff
@@ -573,9 +573,11 @@ private void GetLoadedModules(string[] names, IDictionary<string, ModuleSpecific
             IDictionary<string, ModuleSpecification> moduleSpecTable,
```

Comment:
@TravisEz13, your last commit had 1 failures in `PowerShell-CI-static-analysis`
Verify Markdown Links.Verify links in /home/vsts/work/1/s/docs/building/internals.md.https://powershell.myget.org/feed/powershell-core/package/nuget/PowerShell.Core.Instrumentation should work
```powershell
retry of URL failed with error: 
at <ScriptBlock>, /home/vsts/work/1/s/test/common/markdown/markdown-link.tests.ps1: line 111
111:                                 throw "retry of URL failed with error: $($_.Message)"

```